ReText/tab.py

```diff
@@ -188,3 +189,30 @@ def installFakeVimHandler(self):
 			fakeVimEditor.setQuitAction(self.actionQuit)
 			# TODO: action is bool, really call remove?
 			self.p.actionFakeVimMode.triggered.connect(fakeVimEditor.remove)
+
+
+class ReTextPreview(QTextBrowser):
+	"""
+	When links like [test](test) are clicked, the file test.md is opened.
+	It has to be located next to the current opened file.
+	Relative pathes like [test](../test) or [test](folder/test) are also possible.
+	"""
+
+	def __init__(self, tab):
+		QTextBrowser.__init__(self)
+		self.tab = tab
+		self.setOpenLinks(False)  # if set to True, links to other files will unsuccesfully be opened as anchors
+		self.anchorClicked.connect(self.openInternal)
+
+	def openInternal(self, link):
+		if link.url().startswith('http'):
+			QDesktopServices.openUrl(link)
+		else:
+			currentFileInfo = QFileInfo(self.tab.fileName)
+			current_file_dir = currentFileInfo.absoluteDir().absolutePath()
+			if re.search('\.\w*$', link.url()):  # file suffix exists in link
```

Comment:
This also matches a dot in a directory name, doesn't it? Maybe call `toLocalFile()` first, and then check if it has a suffix?